tests/xbot/common/logic/LatchTest.java

```diff
@@ -14,8 +14,9 @@ public void testRisingEdge() {
         Latch latch = new Latch(false, Latch.EdgeType.RisingEdge);
         MockLatch mockLatch = new MockLatch();
         latch.addObserver(mockLatch);
+        assertEquals(mockLatch.getEdgeType(), null);
         latch.setValue(true);
-        assertTrue(mockLatch.getEdgeType() == Latch.EdgeType.RisingEdge);
+        assertEquals(mockLatch.getEdgeType(), Latch.EdgeType.RisingEdge);
```

Comment:
is it `actual, expected` or `expected, actual`? I think these args might be reversed.